src/index.ts

```diff
@@ -19,7 +19,6 @@ const cli = makeCiCli("sweater-comb", apiCheckService, {
   opticToken: process.env.OPTIC_TOKEN || "",
   gitProvider: {
     token: process.env.GITHUB_TOKEN || "",
-    provider: "github",
   },
```

Comment:
Ah, so we're centralizing this config; this is part of us supporting different gitProviders (like `github` or `gitlab` or `bitbucket`. The reason we can remove this, is because we're storing this config in Optic (part of what we want to do to simplifying onboarding)